packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -0,0 +1,388 @@
+// Disabling because returnTypes must be last param to match 1.x params
+/* eslint-disable default-param-last */
+
+import { EthExecutionAPI, TransactionWithSender } from 'web3-common';
+import { Web3Context } from 'web3-core';
+import {
+	Address,
+	BlockNumberOrTag,
+	convertObjectPropertiesToValidType,
+	convertToValidType,
+	Filter,
+	HexString32Bytes,
+	HexString8Bytes,
+	HexStringBytes,
+	Uint,
+	Uint256,
+	ValidReturnTypes,
+	ValidTypes,
+} from 'web3-utils';
+import { isHexString32Bytes } from 'web3-validator';
+import {
+	convertibleBlockProperties,
+	convertibleFeeHistoryResultProperties,
+	convertibleReceiptInfoProperties,
+	convertibleTransactionInfoProperties,
+} from './convertible_properties';
+
+import * as rpcMethods from './rpc_methods';
+import { BlockFormatted } from './types';
+
+export const getProtocolVersion = async (web3Context: Web3Context<EthExecutionAPI>) =>
+	rpcMethods.getProtocolVersion(web3Context.requestManager);
+
+export const isSyncing = async (web3Context: Web3Context<EthExecutionAPI>) =>
+	rpcMethods.getSyncing(web3Context.requestManager);
+
+export const getCoinbase = async (web3Context: Web3Context<EthExecutionAPI>) =>
+	rpcMethods.getCoinbase(web3Context.requestManager);
+
+export const isMining = async (web3Context: Web3Context<EthExecutionAPI>) =>
+	rpcMethods.getMining(web3Context.requestManager);
+
+export async function getHashRate<ReturnType extends ValidTypes = ValidTypes.HexString>(
+	web3Context: Web3Context<EthExecutionAPI>,
+	returnType?: ReturnType,
+) {
+	const response = await rpcMethods.getHashRate(web3Context.requestManager);
+
+	return convertToValidType(
+		response,
+		returnType ?? web3Context.defaultReturnType,
+	) as ValidReturnTypes[ReturnType];
+}
```

Comment:
If it's all about modifying return type from the RPC method, then why not make it part of the `rpcMethods`?

Reply:
Because the rpc methods are the minimal amount of code to perform the request and adhere to the ETH spec, so should always input/output hex strings